clients/java/src/main/java/io/camunda/zeebe/client/api/command/FailJobCommandStep1.java

```diff
@@ -34,6 +35,18 @@ public interface FailJobCommandStep1 {
   interface FailJobCommandStep2 extends FinalCommandStep<FailJobResponse> {
     // the place for new optional parameters
 
+    /**
+     * Set the backoff timeout for failing this job.
+     *
+     * <p>If the backoff timeout is greater than zero and retries are greater than zero then after
+     * the job this job will be picked up again after this backoff timeout will pass.
```

Comment:
:wrench: We could improve the wording a bit.

```suggestion
     * <p>If the backoff timeout is greater than zero and retries are greater than zero then this
     * job will be picked up again after the given backoff timeout is expired.
```